src/CAREUI/misc/RemainingTime.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+export default function RemainingTime({ time }: { time: number }) {
+  const [remaining, setRemaining] = React.useState(time - new Date().getTime());
+
+  React.useEffect(() => {
+    const interval = setInterval(() => {
+      setRemaining(time - new Date().getTime());
+    }, 1000);
+
+    return () => {
+      clearInterval(interval);
+    };
+  }, [time]);
+
+  return `${Math.max(remaining / 1e3, 0).toFixed(0)}s.`;
+}
```

Comment:
Can we use the `time` returned in `useTimer` hook that was recently added. Something like`${time_to_wait-time}s left`

Reply:
Done